src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Vm-autoShutdown.json

```diff
@@ -0,0 +1,196 @@
+{
+    "name": "Deploy-Vm-autoShutdown",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "mode": "Indexed",
+        "displayName": "Deploy Virtual Machine Auto Shutdown Schedule",
+        "description": "Deploys an auto shutdown schedule to a virtual machine",
+        "metadata": {
+            "version": "1.0.0",
+            "category": "Compute",
+            "source": "https://github.com/Azure/Enterprise-Scale/",
+            "alzCloudEnvironments": [
+                "AzureCloud",
+                "AzureChinaCloud",
+                "AzureUSGovernment"
+            ]
+        },
+        "parameters": {
+            "time": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Scheduled Shutdown Time",
+                    "description": "Daily Scheduled shutdown time. i.e. 2300 = 11:00 PM"
+                },
+                "defaultValue": "0000"
+            },
+            "timeZoneId": {
+                "type": "string",
+                "defaultValue": "UTC",
+                "metadata": {
+                    "displayName": "Time zone",
+                    "description": "The time zone ID (e.g. Pacific Standard time)."
+                }
+            },
```

Comment:
Think we should expose these as input from the portal experiences with some default values, otherwise there are gonna be some surprises for folks

Reply:
@jtracey93 I purposely didn't do this, as the objective is to shutdown running machines that are in subscriptions scheduled to be decommissioned. What value would we add by giving them options to set a time? The objective should be to shutdown machines. What do you think?

Reply:
Will add more context in the portal experience.